include/wx/peninfo.h

```diff
@@ -0,0 +1,145 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/peninfo.h
```

Comment:
The new header needs to be added to `build/files` and `upmake` and `bakefile_gen` rerun, otherwise it won't be installed by `make install`.